tests/Doctrine/Tests/DBAL/Platforms/AbstractSQLServerPlatformTestCase.php

```diff
@@ -656,7 +656,7 @@ public function testCreateTableWithSchemaColumnComments(): void
         $expectedSql = [
             'CREATE TABLE testschema.test (id INT NOT NULL, PRIMARY KEY (id))',
             "EXEC sp_addextendedproperty N'MS_Description', N'This is a comment', "
-                . "N'SCHEMA', 'testschema', N'TABLE', 'test', N'COLUMN', id",
+                . "N'SCHEMA', 'testschema', N'TABLE', 'test', N'COLUMN', 'id'",
```

Comment:
This is also very suspicious. If your PR is a fix, why are you changing existing tests?

Reply:
PR fixes escaping, this is consistent with the title itself. Tests that tested generated SQLs were adjusted, of course, this is needed :)

Reply:
I thought things like table and column names should not be be quoted at all, otherwise they are no longer of the `sysname` datatype. This seems to do the opposite. I'm confused.

Reply:
great - if name will contain `,`, what will happen?

I just added a test to proove, that we need it this way.

Anyway, `sysname` is string datatype - mentioned in that blog post - that is specific in one thing only - defined/optimized to have the same properties as underlaying definition type of identifier, ie. it allows all the characters, supported lengh etc.

Reply:
I can also very easily proove, that the whole DBAL is super buggy when I add test with `.` (I already tried and I just notice how much hot coded this project actually is - check yourself, search for `explode('.', ` - this is of course wrong when does one names like `[table.stilltable.name]` ;-))

Reply:
> great - if name will contain ,, what will happen?

Something bad I suppose… I don't have much time to play riddles. If the solution to the issue is quoting everything that is conceivably quotable, then you should say so in your commit message explicitly. having `id` instead of `'id'` is not broken, there is nothing to fix here. Fixing issue #4283 is a _side effect_ of something you are doing, but for some reason you are refusing to explain what it is you are doing precisely in your commit message.

Reply:
You missed "when used as string literal" from title - this is fix for #4283.